src/templates/addons/aws/index.ts

```diff
@@ -26,19 +33,19 @@ const awsChoices = [
   {
     type: 'input',
     name: 'awsRegion',
-    default: 'ap-southeast-1',
+    default: AWS_DEFAULT_REGION,
     message: 'Which AWS Region do you choose?',
   },
 ];
 
 type AwsOptions = GeneralOptions & {
-  infrastructureType: 'advanced';
-  awsRegion: string;
+  infrastructureType?: 'blank' | 'advanced';
+  awsRegion?: string;
 };
 
-const applyCommonModules = (options: AwsOptions): void => {
-  applyCommon(options);
-  applyRegion(options);
+const applyCommonModules = async (options: AwsOptions) => {
+  await applyTerraformAws(options);
+  await applyAwsRegion(options);
```

Comment:
This function does not apply/copy any module, it copies the provider & set the region. How about `applyProviderAndRegion`

Reply:
Changed in https://github.com/nimblehq/infrastructure-templates/pull/169/commits/f0221ecc22b07b9364d519872160b499d8fb9aff